tools/build_variables.bzl

```diff
@@ -960,3 +964,7 @@ aten_native_source_list = [
     "aten/src/ATen/TensorIterator.cpp",
     "aten/src/ATen/LegacyTHFunctionsCPU.cpp",
 ]
+
+# Files in ATen/native with a few exceptions
+# TODO: move the exceptions to proper locations
+aten_native_source_list = aten_fixed_native_source_list + aten_regenereated_native_source_list_lite
```

Comment:
What's the difference between fixed and regenerated? Could you add comments here?